src/main/resources/org/joda/money/MoneyData.csv

```diff
@@ -23,6 +23,7 @@ BRL,986,2,BR
 BSD,44,2,BS
 BTN,64,2,BT
 BWP,72,2,BW
+BYN,-1,2,BY
```

Comment:
Why it won't load? Is it good idea to comment it out? I think there would be many cases where you might need to construct Money object with BYR currency.

I'll change -1 to 933.